ui/app/AppLayouts/Profile/popups/RenameAccontModal.qml

```diff
@@ -83,9 +83,9 @@ StatusModal {
             anchors.top: selectedColor.bottom
             anchors.topMargin: 10
             anchors.horizontalCenter: parent.horizontalCenter
-            model: Constants.preDefinedWalletAccountColors
+            model: Theme.palette.customisationColorsArray
             titleText: qsTr("color").toUpperCase()
```

Comment:
I know this is not your change, but please, if it really needs to be in uppercase change to `titleText: qsTr("COLOR")`.